packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -502,6 +503,25 @@ class DropdownMenu<T> extends StatefulWidget {
   /// Defaults to [DropdownMenuCloseBehavior.all].
   final DropdownMenuCloseBehavior closeBehavior;
 
+  /// Specifies the maximum number of lines the selected value can display
+  /// in the [DropdownMenu].
+  ///
+  /// If this is 1 (the default), the text will not wrap, but will scroll
+  /// horizontally instead.
+  ///
+  /// If this is null, there is no limit to the number of lines, and the text
+  /// container will start with enough vertical space for one line and
+  /// automatically grow to accommodate additional lines as they are entered, up
+  /// to the height of its constraints.
+  ///
+  /// If this is not null, the value must be greater than zero, and it will lock
```

Comment:
```suggestion
  /// If this is not null, the provided value must be greater than zero. The text field will restrict
```


Avoid using the word "it" in the docs. always be explicit.

Reply:
I took this from TextField only. It is using the word 'it'.

Reply:
Done

Reply:
When updating code and docs, it's good to practice to iterate and improve, if necessary. You'll find this often when working in a large code base with huge of contributions over time. If another does says "it", we should still follow best practices in the new changes.